plugins/labhub.py

```diff
@@ -129,10 +110,18 @@ def invite_cmd(self, msg, match):
 
             # send the invite
             self.TEAMS[team_mapping[team.lower()]].invite(invitee)
-            return self.INVITE_SUCCESS[team.lower()].format(invitee)
+            return tenv().get_template(
+                'promotions/{}.jinja2.md'.format(team.lower())
+            ).render(
+                username=invitee,
+            )
         else:
-            return ('@{}, you are not a maintainer, only maintainers can invite'
-                    ' other people. Nice try :poop:'.format(inviter))
+            return tenv().get_template(
+                'errors/not-maintainer.jinja2.md'
+            ).render(
+                action='invite other people',
+                username=invitee,
```

Comment:
`target=` , or maybe `target_username` or `subject_username` (just ideas; whatever is chosen should be consistently used in all template invocations)